content/02.introduction.md

```diff
@@ -27,6 +27,11 @@ This is a real-time, collaborative effort that welcomes submissions from scienti
 Coronaviruses are RNA viruses that...
 [Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important]
 
+The origin of the SARS-CoV-19 virus is not yet fully understood.
+Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-19 is unlikely to have originated from a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi: 10.1038/s41591-020-0820-9].
+Among known coronaviruses, SARS-CoV-19 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi 10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi 10.1101/2020.02.19.950253].
+This suggests that SARS-CoV-19 may have originated in viral reservoirs of similar hosts, however current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi 10.1038/s41591-020-0820-9].
```

Comment:
```suggestion
This suggests that SARS-CoV-19 may have originated in viral reservoirs of similar hosts, however current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].
```

Reply:
Thanks a lot, I've fixed the tags accordingly.